students/CCSt130/lesson01/front_back.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon May 13 2019
+
+@author: Chuck Stevens :: CCSt130
+"""
+
+varGW = "Strategery"
```

Comment:
Variable names should be snake_case too, and something meaningful (easy to decipher)